packages/bot-web-ui/src/components/dashboard/tutorial-tab/guide-content/guide-content.tsx

```diff
@@ -51,6 +50,13 @@ const GuideContent = observer(({ guide_list }: TGuideContent) => {
         }
     };
 
+    const handleKeyboardEvent = e => {
+        const enterKeyCodes = ['Enter', 'Return', 13];
+        if (enterKeyCodes.includes(e.key) || enterKeyCodes.includes(e.keyCode)) {
```

Comment:
`event.key === 'Enter'` should be enough here. Can you please check this?

Reply:
done ✅